### DISCUSSION TITLE
Is index used where search by Label?

### DISCUSSION THREAD

**Discussion by TouSS** [JanusGraph/janusgraph#3019] [Q&A] [answered]
Hi guys,

  I wondered whether the index is used, when search by Label ?   Such as:
  g.V().hasLabel(label)

version: 0.6.1

Thanks;

**Comment by porunov** [comment, marked as answer]
As for now Label only indices are not supported. That said, a combination of label + key / keys is supported.
At this point you need at least 1 key to create an index.
You can track this issue here: https://github.com/JanusGraph/janusgraph/issues/283
